btc-withdrawal/src/main/kotlin/com/d3/btc/withdrawal/config/BtcWithdrawalConfig.kt

```diff
@@ -13,8 +13,11 @@ interface BtcWithdrawalConfig {
     val signatureCollectorCredential: IrohaCredentialConfig
     // Account that is used to deal with registered accounts
     val registrationCredential: IrohaCredentialConfig
+    //TODO this must be removed
```

Comment:
remove?

Reply:
This part of the config is redundant.  It will be removed soon,